BrowserKit/Sources/ToolbarKit/AddressToolbar/BrowserAddressToolbar.swift

```diff
@@ -247,6 +248,31 @@ public class BrowserAddressToolbar: UIView,
         updateActionStack(stackView: pageActionStack, toolbarElements: state.pageActions)
 
         updateActionSpacing()
+
+        let isAnimationEnabled = !UIAccessibility.isReduceMotionEnabled
+        updateToolbarLayout(animated: isAnimationEnabled)
+    }
+
+    private func updateToolbarLayout(animated: Bool) {
```

Comment:
Wouldn't it make sense to check `UIAccessibility.isReduceMotionEnabled` directly in this method instead of passing `animated`?